modules/tinymce/src/core/main/ts/api/dom/StyleSheetLoader.ts

```diff
@@ -8,9 +8,9 @@
 import { navigator } from '@ephox/dom-globals';
```

Comment:
Should this module also be added to `Tinymce.ts` or do we want to limit it so only we can use it? (ie with the current changes integrators can't use it)

I happy either way, but if we aren't adding it to `Tinymce.ts` then we probably should remove the changelog entry.

Reply:
The intention was to allow integrators to use it - sorry, I wasn't aware of this step.

Reply:
Does this look right? I've exposed the "constructor" function.

Reply:
No, it's in the wrong place, so this needs to be fixed. It should be under `tinymce.dom`, not `tinymce`, as this relates to dom functionality and it's in the `api/dom/` directory.

Also, this should **not** have been merged, as the task list checks weren't completed.